### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [jendrikseipp/rednotebook#617] [Ideas]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by jendrikseipp** [comment]
Nice idea! I currently lack the time to do this myself, but feel free to add RedNotebook to Pacstall.
